.azure-pipelines/azure-pipelines.yml

```diff
@@ -212,6 +227,13 @@ stages:
             - name: RHEL 9.3
               test: rhel/9.3
 
+  # Keep this until the tests work.
+  # It allows us to test against PostgreSQL 10.
+  # PostgreSQL 13 is still supported, but the closest
+  # version we test against is 14.
+  # Keeping testing against 10 and 14 should cover 13.
+  # When these tests stop working and it's hard to fix
+  # or when PostgreSQL 13 gets EOL, remove this target.
```

Comment:
I'd rephrase this, along the lines of: When the tests stop working, remove this target and update the documentation with the fact.

As long as this is tested and works, the documentation can show that.